app/javascript/src/Plans/components/DefaultPlanSelect.jsx

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import React, { useState } from 'react'
+
+import { Select, SelectVariant } from '@patternfly/react-core'
+import { SelectOptionObject, toSelectOption } from 'utilities/patternfly-utils'
+
+import './DefaultPlanSelect.scss'
+
+import type { ApplicationPlan } from 'Types'
+
+type Props = {
+  plan: ApplicationPlan,
+  plans: ApplicationPlan[],
+  onSelectPlan: (ApplicationPlan) => void,
+  isDisabled?: boolean
+}
+
+const DefaultPlanSelect = ({ plan, plans, onSelectPlan, isDisabled = false }: Props) => {
+  // $FlowFixMe Flow is wrong here
```

Comment:
Can you add a note explaining why? It would be good when working with this in the future

Reply:
It's hard to explain, it's some kind of bug and it complains about a prop being `number` instead of `string`, but the type is defined as `number | string`. Not sure how to explain that 😟

Reply:
OK I came up with something in the end